### ISSUE TITLE
Language level and inclusion. 3.1.5

### ISSUE THREAD

**Issue by paul-poupet** [w3c/wcag#2022]
Dear all,

After reading the discussion of @jake-abma and @alastc on reading level :

- The personaes that suffer from a low reading level are not only specific disabled people but also allophones (people whose first language is a foreign language), and some elderly people. The total proportion of the population who is in difficulty with an high writing level is quite high **(16% according to most studies).**
- In many countries, most services, especially public ones, are dematerialized. The level of language is a source of exclusion for these people ​​and raises the question of equality in access to citizen rights.

- Summary, as in technique 86 is not really inclusive, that means the technique is not sufficient.  Those people deserve to be informed, and they can understand all the information of a text if it is well written. One cannot decide for them what part of the information they can access to, this is exclusion.
- The note on E2R is quite true : unfortunately there is no universal language. The same reasoning process on visual illustration shows that technique 103 is not sufficient, those visual explanations will be too specific.
- Clear language (approximately equivalent to lower secondary education) has made its proofs in many countries, with some solid research paper to attest its efficiency (especially in Canada, or in Europe and in the US). It is a standard which is used today by more and more governments and administrations.

-> Anyway, my point is that **technique 153 is the only sufficient technique**. 103 and 86 cannot be considered as alternatives techniques but they are techniques to go further after the application of technique 153
-> Moreover, a **binary result on the clarity of the language can be defined**. Global functions (ratio of words/sentences…) doesn’t seem to fit, but a part of the important things are listed in the description of G153, it just has to be completed and turned into an algorithm with a binary answer (i can work on it with some researchers in linguistics, somepeople already implemented reading level scores of a text : https://cental.uclouvain.be/amesure/ in belgium, https://u31.io in france...). 

=> **Concretely,  153 is the main technique, or even better : the use of an understandable language as a AA guideline (either as a primary language or as another language “simplified english”)**, and all the other techniques (86 in supplement, 103, or the idea of including disabled peoples in a creation or validation process of the content) in a AAA guideline.

_16% of the population, without other disabilities, has difficulty reading and understanding written information: making written content more readable is necessary for all of these people._

I can make a pull request if other experts do not have concerns

Thanks for reading, i’ll be happy to discuss on those points (reach me on github or at paul.poupet@seed-up.io
Paul
